controllers/app.controllers.js

```diff
@@ -13,4 +13,20 @@ exports.getEndpoints = (req, res, next) => {
     .then((endpoints) => res.status(200).send({ endpoints }))
     .catch((err) => {
     })
+}
+
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    fetchArticleById(article_id)
+    .then((article) => {
+        if (!article.length) {
+            return Promise.reject({
+                status: 400,
+                msg: "No article found with that id"
+            })
```

Comment:
Ideally, this error handler should be in the model. It should also be a 404, not 400